astropy/io/fits/tests/test_connect.py

```diff
@@ -686,7 +687,10 @@ def assert_objects_equal(obj1, obj2, attrs, compare_class=True):
             if a2 is None:
                 a2 = {}
 
-        assert np.all(a1 == a2)
+        if isinstance(a1, np.ndarray) and a1.dtype.kind == 'f':
+            assert quantity_allclose(a1, a2, rtol=1e-10)
```

Comment:
Presumably, these are small rounding errors, in which case `rtol=1e-15` may be more appropriate

Reply:
The value I used was the same as in the analogous ECSV test, but it looks like using a smaller value works just fine everywhere.